app/views/dogs/_manager_dog.html.erb

```diff
@@ -1,9 +1,10 @@
-<tr>
-  <td><%= dog.tracking_id %></td>
-  <td><%= link_to dog.name.titleize, dog %></td>
-  <td><%= dog.intake_dt.strftime("%m/%d/%Y") unless dog.intake_dt.blank? %></td>
-  <td><%= dog.age ? dog.age.titleize : " " %></td>
-  <td><%= dog.size ? dog.size.titleize : " " %></td>
+<tr class='dog'>
+  <td class='tracking_id'><%= dog.tracking_id %></td>
+  <td class='name'><%= link_to dog.name.titleize, dog %></td>
+  <td class='breed'><%= dog.breeds.join("</br>").html_safe %></t>
```

Comment:
hakiri flags line 4 as possible xss vulnerability. However dog.breeds is the concatenation of primary and secondary breed names from the database, it's not user-entered.

Reply:
this one is so weird in Hakiri